src/assets/css/blogposts.scss

```diff
@@ -12,9 +13,38 @@ $pull-quote-color: #4e1a6a;
   }
 }
 
+.blogpost-breadcrumb {
```

Comment:
Does this need to be `display: grid`? It doesn't seem like it's fits the mental model of a grid to me but maybe I'm missing something?

Reply:
I am not sure to follow. The breadcrumb is using the first row of the main content grid:

![Screen Shot 2021-04-26 at 12 08 55](https://user-images.githubusercontent.com/217628/116066320-37641100-a688-11eb-93e7-dbdeafe10883.png)

Reply:
(screenshot taken while working on the `nav` approach)

Reply:
Ah ok, that's the bit I was missing, thanks.